animation_nodes/nodes/spline/splines_from_edges.py

```diff
@@ -1,10 +1,14 @@
 import bpy
 from bpy.props import *
-from . c_utils import splinesFromEdges
 from ... events import propertyChanged
-from ... data_structures import DoubleList
+from ... data_structures import VirtualDoubleList
 from ... base_types import AnimationNode, VectorizedSocket
+from . splines_from_edges_utils import splinesFromBranches, splinesFromEdges
 
+algorithmItems = [
```

Comment:
just by convention either this should be called `algorithmTypeItems` or the property in the node should just be called `algorithm`.